cpp/cugraph_cmake/cugraph.cmake

```diff
@@ -108,6 +110,6 @@ macro(target_mage_cugraph target_name)
       PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:${MAGE_CXX_FLAGS}>"
               "$<$<COMPILE_LANGUAGE:CUDA>:${MAGE_CUDA_FLAGS}>"
     )
-    target_link_libraries("${target_name}" PRIVATE mage_cugraph)
+    target_link_libraries("${target_name}" PRIVATE mage_cugraph CUDA::toolkit)
```

Comment:
The `CUDA::toolkit` here probably should be deleted here because it's already included.